js/src/View/TodoView.js

```diff
@@ -0,0 +1,53 @@
+var Backbone = require('backbone');
+var $ = require('jquery');
+var _ = require('underscore');
+var handlebars = require('handlebars');
+
+var TodoCollection = require('../Collection/TodoCollection.js');
+var TodoModel = require('../Model/TodoModel.js');
```

Comment:
Surprisingly, we don't actually use `TodoCollection` or `TodoModel` in this file. I would be sure to add that the expected model is a `TodoModel`, however, in either the class or constructor comment.